client-tools/Dump/arangodump.cpp

```diff
@@ -67,7 +67,7 @@ int main(int argc, char* argv[]) {
             argv[0], "Usage: arangodump [<options>]",
             "For more information use:", BIN_DIRECTORY));
     ApplicationServer server(options, BIN_DIRECTORY);
-    int ret;
+    int ret = EXIT_SUCCESS;
```

Comment:
Initialization is always good.

Reply:
But... why do we only initialize in arangodump and not the other client tools? If we fix it here, we should fix it in the other client tools as well.